src/ucs/debug/memtrack.c

```diff
@@ -172,6 +172,13 @@ void ucs_memtrack_releasing(void* ptr)
 void *ucs_malloc(size_t size, const char *name)
 {
     void *ptr = malloc(size);
+
+#ifdef UCX_ALLOC_ALIGN
```

Comment:
how about adding this to ucs_memtrack_allocated() so it will check all allocations, not just malloc?